core/src/main/java/com/amazon/opendistroforelasticsearch/sql/data/model/ExprValueUtils.java

```diff
@@ -172,4 +171,8 @@ public static Boolean getBooleanValue(ExprValue exprValue) {
   public static ZonedDateTime getDateValue(ExprValue exprValue) {
     return exprValue.dateValue();
   }
+
+  public static TemporalAmount getIntervalValue(ExprValue exprValue) {
```

Comment:
do we need this method?

Reply:
yes, i've also added getTimeValue, getDatetimeValue, getTimestampValue in the other PR #694  to stay consistent. We can also remove all these methods including the getDateValue actually